### ISSUE TITLE
About page: Links to open data toolkit should open a new tab

### ISSUE THREAD

**Issue by tkeuneman** [magda-io/magda#1463]
### Problem description
In the About page content, links to open data toolkit should open a new tab. This is a DGA pattern, where a link to an external website should open in a new tab. Especially useful for mobile.

**Comment by stephencannings** [comment]
WCAG discourages opening new windows as users with sight issues or those using assistive devices can become disorientated. 

G200: Opening new windows and tabs from a link only when necessary
https://www.w3.org/TR/WCAG20-TECHS/G200

**Comment by aneesha09** [comment]
In this case, I think it is necessary as it is linking to an external site.

**Comment by sandra-arato** [comment]
Hi there, 

I'd like to help out with this issue. I agree that assistive technology users would lose context if link is opened in a new window, unless they are aware of this in advance. So I propose to make the external links open in a new window, but have a visually hidden label after the link text that notifies screenreader users about the target.

I think the [G201 Guideline on the WCAG](https://www.w3.org/TR/WCAG20-TECHS/G201.html) docs is a well-fitting pattern here, and I've created a [codepen about my proposed change](https://codepen.io/sandraszenti/pen/xyVrmK).

Can I make the suggested change?

Also, as this is my first contribution, can you please help out with where I find the content in the repo? I don't seem to find the markdown file for the About page.

**Comment by AlexGilleran** [comment]
@sandra-arato Awesome! That sounds good - the markdown file is here: https://github.com/magda-io/magda/blob/master/magda-web-client/src/markdown/about.md.

I _think_ it's set to render HTML tags in the markdown so it should be OK, if not we might need to fiddle with the rendering code.

This should just be a front-end modification, so you can just build/run the front end without having to set up Kubernetes: https://github.com/magda-io/magda/blob/master/docs/docs/building-and-running.md#building-and-running-just-the-frontend

**Comment by sandra-arato** [comment]
Hi @AlexGilleran , 
I've made a PR (see above) - however [my build failed](https://travis-ci.org/magda-io/magda/jobs/437400982) with a few ```index_not_found``` exceptions. Is it possible that it's caused because I issued the PR from the forked repo, and not the original one? The CONTRUBING.md suggested to work on a forked version. 

I also noticed that I don't have the option to assign my PR to anyone on the first PR (that's what I checked with the second, closed PR).

Thank you 😄
